weblab/entities/tests/test_views.py

```diff
@@ -842,6 +842,42 @@ def test_lists_my_protocols(self, client, logged_in_user):
         assert response.status_code == 200
         assert list(response.context['object_list']) == protocols
 
+@pytest.mark.django_db
+class TestTransfer:
+    def test_transfer(self, client, logged_in_user, other_user, helpers):
+        helpers.add_permission(logged_in_user, 'create_model')
+        model = recipes.model.make(author=logged_in_user)
+        commit = helpers.add_version(model, visibility='public')
+
+        assert model.author.email == 'test@example.com'
+        response = client.post(
+            '/entities/models/%d/transfer' % model.pk,
+            data={
+                'email': other_user.email,
+            },
+            )
+        assert response.status_code == 302
+        model2 = ModelEntity.objects.first()
+        assert model2.author.email == 'other@example.com'
```

Comment:
I'd check `assert model.author == other_user` to make the connection more explicit.